src/components/HistoryTables/MessageHistory/History/MessagePendingRow.tsx

```diff
@@ -34,7 +39,7 @@ export default function PendingMessageHistoryRow(
           {truncateAddress(message.cid)}
         </SmartLink>
       </td>
-      {props.inspectingAddress && <td>{methodName}</td>}
+      <td>{methodName}</td>
```

Comment:
I'm fine taking this out, but what prompted this change? Why did it happen in this PR? Just curious

Reply:
Same reason as the other occurence: not necessary to check for `props.inspectingAddress` and we don't want to render the `td` conditionally to prevent content shifting to the wrong column.